scalafix/input/src/main/scala/fix/CanBuildFromSrc.scala

```diff
@@ -33,4 +33,11 @@ object CanBuildFromSrc {
     cbf2("")
     ()
   }
+
+  def f2[T, That](implicit cbf: CanBuildFrom[Nothing, T, That]): Foo[T, That] =
+    new Foo
+
+  class Foo[T, That](implicit cbf: CanBuildFrom[Nothing, T, That]) {
```

Comment:
We now support Classes and Def. I don't see another place where we could insert a CanBuildFrom. Maybe on a type alias (`type  Foo = CanBuildFrom[Bar, Buz, Qux]`)?